src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -180,21 +180,52 @@ function validatePropTypes(element) {
   }
 }
 
+function warnInvalidType(inputValue) {
+  // warning(...) messages need to be string literals.
+  // If not, we can do a ternary expression to evaluate the need
+  // for the mistyped import message, and not repeat the message
+  // twice here.
+  if (inputValue === undefined) {
+    warning(
+      false,
+      'React.createElement: %s is an invalid value for a type. ' +
+      'It should be a string (for DOM elements), ReactClass or '+
+      'React.Component (for composite components). ' +
+      'Did you mistype an import or forget to export your component?' +
```

Comment:
I want to move this sentence right after the first sentence. It’s easier to miss and ignore at the end.